auraed/src/runtime/cells/mod.rs

```diff
@@ -1,15 +1,16 @@
 use cell::Cell;
 use cell_name::CellName;
 pub(crate) use cell_service::CellService;
-use cells_table::CellsTable;
+use cells::Cells;
 use error::{CellsError, Result};
 use executable::Executable;
 use executable_name::ExecutableName;
 
 mod cell;
 mod cell_name;
 mod cell_service;
-mod cells_table;
+#[allow(clippy::module_inception)]
+mod cells;
```

Comment:
maybe cell_cache? :)

or cell_impl? (ew).

Reply:
I think `Cells` is the best name for it (maybe `CellsCollection`? if we wanted to change it) and the clippy warning is just a result of keeping the `cells` module in `auraed` instead of its own crate, which I don't think in necessary yet.

I moved away from 'CellsTable' because to the user of `Cells`, its implementation is unimportant at this point. 'CellsTable' or 'CellsCache' feels like it's leaking the "unimportant" details.

Reply:
CellsController?

Reply:
it's fine as it is.